packages/desktop-client/src/components/mobile/MobileAmountInput.js

```diff
@@ -0,0 +1,399 @@
+import { PureComponent, createContext, forwardRef } from 'react';
+
+import mitt from 'mitt';
+
+import {
+  toRelaxedNumber,
+  amountToCurrency,
+  getNumberFormat,
+} from 'loot-core/src/shared/util';
+
+import { colors } from '../../style';
+import { Button, Text, View } from '../common';
+
+function getValue(state) {
+  const { value } = state;
+  return value;
+}
+
+const AmountAccessoryContext = createContext(mitt());
```

Comment:
It looks like this is unnecessary. The `reset` event has no listeners, and the `done` event appears to never be emitted.

Reply:
Good catch, removed in https://github.com/actualbudget/actual/pull/1340/commits/71c29a034449467fbee9029bbd03e22be7529f55